lib/diagnostic.js

```diff
@@ -39,20 +52,30 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Marmaduke',
+  species: 'Dog',
+  breed: 'Siberian Husky',
+  vocalization: 'Gentle',
+};
```

Comment:
I think you forgot to add a method here that returns your `vocalization` property. Something like this would work: `vocalize: function() {return this.vocalize}`